.gitignore

```diff
@@ -36,6 +36,5 @@ out/
 ### VS Code ###
 .vscode/
 
-### Creds ###
-.token
-.creds
```

Comment:
Why have we removed creds?

Reply:
No such file is present in our codebase.